crt/CMakeLists.txt

```diff
@@ -28,7 +28,11 @@ if(UNIX AND NOT APPLE)
     set(DISABLE_GO ON) # Build without using Go, we don't want the extra dependency
     set(DISABLE_PERL ON) # Build without using Perl, we don't want the extra dependency
     set(BUILD_LIBSSL OFF) # Don't need libssl, only need libcrypto
```

Comment:
move these `set()` calls inside the `if (NOT USE_OPENSSL)`

they exist to customize how aws-lc is built